File: Retos Abril/Java/Reto2.java
package RetosAbril;

import java.util.HashMap;
import java.util.LinkedHashMap;

public class Reto2 {
    public static void main(String[] args) {
        HashMap<String, Integer> calendario = new LinkedHashMap<>();
        calendario.put("Enero", 31);
        calendario.put("Febrero", 28);
        calendario.put("Marzo", 31);
        calendario.put("Abril", 30);
        calendario.put("Mayo", 31);
        calendario.put("Junio", 30);
        calendario.put("Julio", 31);
        calendario.put("Agosto", 31);
        calendario.put("Septiembre", 30);
        calendario.put("Octubre", 31);
        calendario.put("Noviembre", 30);
        calendario.put("Diciembre", 31);

        System.out.println("***** Calendario *****");
        for (Object mes : calendario.keySet()) {
            System.out.print(mes + ": \n");
            for (int i = 1; i < calendario.get(mes) + 1; i++) {
                if (i % 7 == 0) {
                    System.out.println(i + "\t");
                } else {
                    System.out.print(i + "\t");
                }
            }
            System.out.println("\n");
        }
    }
}

File: Retos Abril/Python/Reto_2.py
calendario = {
    "Enero": 31,
    "Febrero": 28,
    "Marzo": 31,
    "Abril": 30,
    "Mayo": 31,
    "Junio": 30,
    "Julio": 31,
    "Agosto": 31,
    "Septiembre": 30,
    "Octubre": 31,
    "Noviembre": 30,
    "Diciembre": 31
}

for mes, dias in calendario.items():
    print("\n" + mes, end= ": ")
    for dia in range(1, dias + 1):
        print(dia, end= " ")
